template/default.tmpl

```diff
@@ -31,6 +31,11 @@
 {{ define "pagerduty.default.instances" }}{{ template "__text_alert_list" . }}{{ end }}
 
 
+{{ define "victorops.default.entity_id" }}{{ .GroupLabels.SortedPairs.Values | join " " }} {{ if gt (len .CommonLabels) (len .GroupLabels) }} / {{ with .CommonLabels.Remove .GroupLabels.Names }}{{ .Values | join " " }}{{ end }}{{ end }}{{ end }}
```

Comment:
It's incorrect to include the common labels, this should only be the group labels.